src/plugins/discover/public/application/angular/doc_table/lib/get_default_sort.ts

```diff
@@ -32,6 +31,6 @@ export function getDefaultSort(
   if (indexPattern.timeFieldName && isSortable(indexPattern.timeFieldName, indexPattern)) {
     return [[indexPattern.timeFieldName, defaultSortOrder]];
   } else {
-    return [['_score', defaultSortOrder]];
```

Comment:
quick question @timroes , I don't think it makes much sense to sort by `_score` by default when it's an index pattern without timefield. this function now returna an empty default sorting, which would mean, that ES get's the info sort by `_doc`. I think that's a better choice here, would you agree?

Reply:
@timroes update: changed the default to `_score` for index patterns without time field (so it is handled like before and nothing breaks)